### ISSUE TITLE
CVE-2020-24307 and CVE-2023-30367

### ISSUE THREAD

**Issue by etug80** [mRemoteNG/mRemoteNG#2585]
Hello Folks!

I'm glad to come across mRemoteNG when I searched for Libre Software alternative to SecureCRT.

Unfortunately, when I requested my organization to install mRemoteNG (either v1.76.20 or v1.77.3.1784-NB) on my work laptop, the SOC team rejected my request citing the following CVEs.
[1] https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-24307
[2] https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-30367

Any inside information regarding these?

P.S.
Thank you all for making this software.

**Comment by Kvarkas** [comment]
yes, issue exist and currently working to eliminate in new 1.77.3 realize, what require a big code chuck to change so a bit slow with release...

**Comment by JeanPluzo** [comment]
Hi,

NIST just posted an official statement regarding mRemoteNG a few weeks ago:
https://nvd.nist.gov/vuln/detail/CVE-2023-30367
btw, we have to stop using mRemoteNG at work because of this. It's sad that there are no new builds.
Linking [this](#2474) thread yet again.

Regards,
J.

**Comment by Avrution** [comment]
Still hoping for a new build

**Comment by CptnFizzbin** [comment]
I think you meant to link #2474 there @JeanPluzo

**Comment by robertpopa22** [comment]
Triage (P3 stale status): this issue is labeled as actively worked, but activity appears stale. Please confirm whether work is still in progress. If not, please relabel to Need 2 check (or close) to keep status labels accurate.

**Comment by robertpopa22** [comment]
**Fix available for testing**

**Commit:** `f34c13e7` (fork `robertpopa22/mRemoteNG`, branch `main`)
**What changed:** CVE-2020-24307 and CVE-2023-30367

A beta build (v1.81.0-beta.4) including this fix is available from the fork's Releases page.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
